GiniVision/Classes/SupportedFormatsViewController.swift

```diff
@@ -0,0 +1,116 @@
+    
+    let reuseIdentifier = "reuseIdentifier"
+    let rowHeight: CGFloat = 70
+    let sectionHeight: CGFloat = 70
+    var sections: [(title: String, items: [String], itemsImage:UIImage?, itemsImageBackgroundColor: UIColor)] = [
+        ("Folgende Formate werden unterstützt:",
+         ["Computer-erstellte Überweisungsträger und Rechnungen",
+          "Einseitige Bilder im jpg, png oder tif Format",
+          "PDF Dokumente von bis zu 10 Seiten"],
+         UIImage(named: "supportedFormatsIcon", in: Bundle(for: GiniVision.self), compatibleWith: nil),
+         GiniConfiguration.sharedConfiguration.supportedFormatsIconColor),
+        ("Was nicht analysiert wird:",
+         ["Handschrift",
+          "Fotos von Bildschirme"],
+         UIImage(named: "nonSupportedFormatsIcon", in: Bundle(for: GiniVision.self), compatibleWith: nil),
+         GiniConfiguration.sharedConfiguration.nonSupportedFormatsIconColor)
+    ]
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        title = "Unterstützte Formate"
+        tableView.register(SupportedFormatsTableViewCell.self, forCellReuseIdentifier: reuseIdentifier)
+        tableView.rowHeight = rowHeight
+        tableView.tableFooterView = UIView()
+        tableView.sectionHeaderHeight = sectionHeight
+        tableView.allowsSelection = false
+        tableView.backgroundColor = Colors.Gini.pearl
+        tableView.alwaysBounceVertical = false
+        
+        if #available(iOS 11.0, *) { // On iOS is .automatic by default.
+            tableView.contentInsetAdjustmentBehavior = .never
+        }
+    }
+    
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return sections.count
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return sections[section].items.count
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let section = sections[indexPath.section]
+        let item = section.items[indexPath.row]
+        
+        let cell = tableView.dequeueReusableCell(withIdentifier: reuseIdentifier, for: indexPath) as! SupportedFormatsTableViewCell
+        cell.textLabel?.text = item
+        cell.imageView?.image = section.itemsImage
+        cell.imageBackgroundView.backgroundColor = section.itemsImageBackgroundColor
+
+        return cell
+    }
+    
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        return sections[section].title
+    }
+    
+    override func tableView(_ tableView: UITableView, willDisplayHeaderView view: UIView, forSection section: Int) {
+        if let header = view as? UITableViewHeaderFooterView {
+            header.backgroundView?.backgroundColor = nil
+        }
+    }
+
+}
+
+final class SupportedFormatsTableViewCell: UITableViewCell {
```

Comment:
I think `SupportedFormatsTableViewCell ` is getting too big for being in the same file. If it was just a few lines of code, leaving it in the view controller would have been fine. But in this particular case, I'd put it in a new file.

Reply:
Yes, but I think we should start creating folders inside GiniVision project to keep it oganized.